src/sentry/runner/commands/devserver.py

```diff
@@ -129,6 +129,7 @@ def _get_daemon(name: str) -> tuple[str, list[str]]:
     envvar="SENTRY_DEVSERVER_BIND",
     required=False,
 )
+@click.option("--enable-dlq", "dlq_consumers", multiple=True, help="The consumer to enable DLQ for")
```

Comment:
What does this actually do for consumers that don't have DLQ? I'd prefer not to have this and have devserver always fail

Reply:
Fails with `click.BadParameter`
```
Error: Invalid value: Cannot enable DLQ for consumer: ingest-events, DLQ topic is not specified in consumer definition
```